sections/shipments.md

```diff
@@ -99,8 +99,8 @@ _Optional parameters_:
     "insurance_fee": null,
     "delivery_fee": "1.00",
     "created_at": "2017-11-14T11:04:32.168-08:00",
-    "postage_label_png_url": "https://chitchats.com/labels/shipments/8ernj8scq9.png",
-    "postage_label_zpl_url": "https://chitchats.com/labels/shipments/8ernj8scq9.zpl"
+    "postage_label_png_url": "https://chitchats.com/labels/shipments/8ernj8scq9.png?...",
```

Comment:
Wondering if we can include the `auth_token` param for more clarity, considering it's the only attribute included for now

```suggestion
    "postage_label_png_url": "https://chitchats.com/labels/shipments/8ernj8scq9.png?auth_token=...",
```

Reply:
I kept it consistent with how we display batch response. Just kept it as short as possible. Technically I don't think they need to know the actual params.

`"https://chitchats.com/labels/client_batches/2.png?..."`